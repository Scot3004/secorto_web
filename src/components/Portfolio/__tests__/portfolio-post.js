import React from "react"
import renderer from "react-test-renderer"
import PortfolioPost from "../portfolio-post"

const gallery = [
  {
    image: {
      thumbnail: "img1",
    },
    alt: "img1",
  },
  {
    image: {
      thumbnail: "img1",
    },
    alt: "img2",
  },
]

describe("PortfolioPost", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(
        <PortfolioPost
          title="portfolio-post"
          role="my-role"
          responsibilities="my-responsibilities"
          gallery={gallery}
          image="testImage"
        >
          content
        </PortfolioPost>
      )
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
