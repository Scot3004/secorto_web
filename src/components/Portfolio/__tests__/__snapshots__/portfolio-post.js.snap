// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PortfolioPost renders correctly 1`] = `
.emotion-0 {
  padding: 1rem;
}

.emotion-1 {
  color: primary;
  margin-top: 16px;
}

.emotion-2 {
  display: inline-block;
  vertical-align: middle;
  overflow: hidden;
  position: relative;
  top: -2px;
}

.emotion-6 {
  margin: 0.5rem;
}

<main>
  <img
    alt="portfolio-post"
    src="testImage"
  />
  <div
    className="emotion-0"
  >
    <h1
      className="emotion-1"
    >
      portfolio-post
    </h1>
    <p>
      <svg
        aria-hidden="true"
        className="emotion-2 emotion-3"
        fill="currentColor"
        focusable="false"
        height={24}
        viewBox="0 0 24 24"
        width={24}
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M0 0h24v24H0V0z"
          fill="none"
        />
        <path
          d="M12 16c-2.69 0-5.77 1.28-6 2h12c-.2-.71-3.3-2-6-2z"
          opacity={0.3}
        />
        <circle
          cx={12}
          cy={8}
          opacity={0.3}
          r={2}
        />
        <path
          d="M12 14c-2.67 0-8 1.34-8 4v2h16v-2c0-2.66-5.33-4-8-4zm-6 4c.22-.72 3.31-2 6-2 2.7 0 5.8 1.29 6 2H6zm6-6c2.21 0 4-1.79 4-4s-1.79-4-4-4-4 1.79-4 4 1.79 4 4 4zm0-6c1.1 0 2 .9 2 2s-.9 2-2 2-2-.9-2-2 .9-2 2-2z"
        />
      </svg>
      my-role
    </p>
    <p>
      <svg
        aria-hidden="true"
        className="emotion-2 emotion-3"
        fill="currentColor"
        focusable="false"
        height={24}
        viewBox="0 0 24 24"
        width={24}
        xmlns="http://www.w3.org/2000/svg"
      >
        <rect
          fill="none"
          height={24}
          width={24}
        />
        <rect
          height={2}
          width={11}
          x={3}
          y={10}
        />
        <rect
          height={2}
          width={11}
          x={3}
          y={6}
        />
        <rect
          height={2}
          width={7}
          x={3}
          y={14}
        />
        <polygon
          points="20.59,11.93 16.34,16.17 14.22,14.05 12.81,15.46 16.34,19 22,13.34"
        />
      </svg>
      my-responsibilities
    </p>
    content
    <div
      className="pswp-gallery"
      id="galeria-portafolio"
    >
      <a
        className="emotion-6"
        href="testSrc"
        rel="noreferrer"
        target="_blank"
      >
        <img
          alt="img1"
          src="img1"
        />
      </a>
      <a
        className="emotion-6"
        href="testSrc"
        rel="noreferrer"
        target="_blank"
      >
        <img
          alt="img2"
          src="img1"
        />
      </a>
    </div>
  </div>
</main>
`;
